tools/docker-network/docker-compose.yml

```diff
@@ -175,7 +177,7 @@ services:
       - "8000/tcp"
       - "8081/tcp"
     deploy:
-      replicas: ${DRNG_REPLICAS:-2}
+      replicas: ${DRNG_REPLICAS:-4}
```

Comment:
Why is suddenly a larger number of replicas needed to do the drand tests, just because we changed the deficit computation?

Reply:
It was because this branch was synced with develop and target branch wasn't. Updated it and it's not a diff in this PR anymore.